applications/chronicle.md

```diff
@@ -0,0 +1,204 @@
+about 3.5 years. It is the tool that allows application developers to
+receive real-time updates from the blockchain, and maintain the
+history of their transactions.
+
+There are several other software solutions for reading the state
+history. But they are either too specific for a particular task, or
+not fast enough for bulk processing. Chronicle offers a universal
+solution, and it is currently the fastest way to decode the state
+history.
+
+## Team
+
+### Team members
+
+- **Team Leader:** CTO of EOS Amsterdam and Senior Consultant at
+    Zaisan.io, working under "cc32d9" pseudonym.
+
+- Daniel Liven, managing the legal and financial aspects.
+
+### Legal Structure
+- **Registered Legal Entity:** Zaisan BV
+- **Registered Address:** Keizersgracht 391A, 1016 EJ, Amsterdam, The Netherlands
+
+### Team Experience
+
+Zaisan has been around in EOSIO ecosystem for several years, formally
+known as Europechain BV. The company is founded by several European
+block producers, and is focusing on software projects and business
+consultancy.
+
+The company has previously taken the task of writing the EOSIO Core+
+Blue Paper on request of ENF. 
+
+### Team Org Repos
+
+- https://github.com/Europechain
+- https://github.com/eos-amsterdam-rnd
+
+### Team Member Repos
+
+- https://github.com/cc32d9
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/dliven/
+
+## Development Status
+
+- [Initial proposal and crowdfunding proceedings](https://github.com/cc32d9/eos-work-proposals/tree/master/001_EOS_Chronicle)
+- [Main project repository](https://github.com/EOSChronicleProject/eos-chronicle)
+- [Chronicle tutorial](https://github.com/EOSChronicleProject/chronicle-tutorial)
+- [JavaScript NPM for Chronicle consumers](https://github.com/EOSChronicleProject/chronicle-consumer-npm)
+- Blog publications: [fundraising](https://cc32d9.medium.com/fundraising-for-chronicle-and-history-indexer-26e0a06c2d1d), [overview of history solutions](https://cc32d9.medium.com/history-and-notifications-in-eosio-blockchain-8255194af93)
+
+## Development Roadmap
+
+### Overview
+
+- **Total Estimated Duration:**  4 months
+- **Full-Time Equivalent (FTE):**  0.25
+- **Total Costs:** USD 20,000
+
+### Milestone 1 — Chronicle software update
+
+- **Estimated duration:** 2 months
+- **FTE:**  0.25
+- **Costs:** 8,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | Apache 2.0 |
+| 0b. | Documentation | Updated Chronicle README and the Tutorial to reflect all the recent changes |
+| 0c. | Testing Guide | Tests scripts and guides will be part of documentation |
+| 1. | Update dependencies | Update Chronicle to work with latest releases of dependency libraries (Boost.org and EOSIO/Mandel libraries).|  
+| 2. | Ubuntu packages | Crate package build scripts; publish the packages for public download |   
+| 3. | JavaScript NPM | Redesign the asynchronous mode to avoid memory overflow |  
```

Comment:
What exactly does this mean?

Reply:
1. update the code and git submodules to work with the current appbase/abieos/fc, and current Boost. Last time I trid, there was no simple upgrade path, so it needs work.
2. there's no packaging as of now, so we have to build from sources every time. Needs work to make proper deb package scripts
3. the NPM is overflowing the nodejs memory in asynchronous mode, as it cannot pause the source flow. So it needs a redesign.

Reply:
Gotcha. Also sorry about that I should've been more explicit. I just meant the NPM part.